sklearn/feature_selection/rfe.py

```diff
@@ -184,14 +190,20 @@ def _fit(self, X, y, step_score=None):
             estimator.fit(X[:, features], y)
 
             # Get coefs
-            if hasattr(estimator, 'coef_'):
-                coefs = estimator.coef_
+            if callable(self.importance_getter):
+                coefs = self.importance_getter(estimator)
+            elif self.importance_getter=='auto':    
```

Comment:
Spaces around == please

Reply:
Again sorry for this formatting mistake.